src/bun.js/javascript.zig

```diff
@@ -1673,7 +1673,7 @@ pub const VirtualMachine = struct {
                     printed,
                 ),
             };
-            res.* = ErrorableString.err(error.NameTooLong, ResolveMessage.create(global, VirtualMachine.get().allocator, msg, source.utf8()).asVoid());
+            res.* = ErrorableString.err(error.NameTooLong, ResolveMessage.create(global, VirtualMachine.get().allocator, msg, source_utf8.slice()).asVoid());
```

Comment:
this also solves an issue on linux when running `test/js/bun/resolve/resolve.test.ts`. I am not sure if this is the right approach